react-blog/src/pages/Home/index.tsx

```diff
@@ -16,7 +16,7 @@ const Home = () => {
     error,
   } = useSelector((state: any) => state.posts);
 
-  const dispatch = useAppDispatch();
+  const dispatch = useDispatch();
```

Comment:
I think u have error here. We used useAppDispatch because redux-toolkit require it

Reply:
ok, thanks, corrected